install_with_docker.sh

```diff
@@ -209,52 +206,34 @@ https://api.segment.io/v1/track \
   }
 }'
 
-INITIAL_PORT=5122
-START_PORT=5122
-END_PORT=5122
+PORT=5122
 
 check_ports() {
-
-    if lsof -Pi :$INITIAL_PORT -sTCP:LISTEN -t > /dev/null || docker ps --format "{{.Ports}}" | grep -q "0.0.0.0:${INITIAL_PORT}->"; then
-        for port in $(seq $START_PORT $END_PORT); do
-            if lsof -Pi :$port -sTCP:LISTEN -t > /dev/null || docker ps --format "{{.Ports}}" | grep -q "0.0.0.0:$port->"; then
-                continue
-            else
-                echo $port
-                return 0
+    if lsof -Pi :$PORT -sTCP:LISTEN -t > /dev/null || docker ps --format "{{.Ports}}" | grep -q "0.0.0.0:${PORT}->"; then
+        CONTAINER_ID=$(docker ps --format "{{.ID}}:{{.Image}}:{{.Ports}}" | grep "siglens/siglens.*0.0.0.0:${PORT}" | cut -d ":" -f 1 2>/dev/null)
+        if [ -n "$CONTAINER_ID" ]; then
+            docker stop $CONTAINER_ID
+            if lsof -Pi :$PORT -sTCP:LISTEN -t > /dev/null || docker ps --format "{{.Ports}}" | grep -q "0.0.0.0:${PORT}->"; then
+                print_error_and_exit "\nError: Port ${PORT} is already in use."
             fi
-        done
-        echo "-1"
-        return 1
-    else
-        echo "${INITIAL_PORT}"
-        return 0
+        else
+            print_error_and_exit "\nError: Port ${PORT} is already in use."
+        fi
     fi
+    print_success_message "\nStarting Siglens on Port ${PORT}"
 }
 
-UI_PORT=$(check_ports)
-
-if [ ${UI_PORT} == "-1" ]; then
-    echo ""
-    tput bold
-    printf "${RED_TEXT}Error: Port ${INITIAL_PORT} is already in use.\n"
-    tput sgr0
-    exit 1
-fi
+check_ports
```

Comment:
we should call check_ports once with 5122 port and next with 8081. Which means we should change check_ports to accept a port as a parameter.
This can be addressed in the next PR